wagtail/search/utils.py

```diff
@@ -179,3 +180,8 @@ def get_search_fields(search_fields):
         elif isinstance(search_field, RelatedFields):
             for sub_field in get_search_fields(search_field.fields):
                 yield sub_field
+
+
+def get_postgresql_connections():
```

Comment:
Looks like this is unused

Reply:
It's used in some tests (`wagtail.search.tests.test_backends.TestBackendLoader.test_get_search_backends_with_no_default_defined, for example`)